src/Link/Link.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+import { LinkProps } from './Link.types';
+
+/** The Link component is an element for navigation. */
+const Link: FC<LinkProps> = ({
+  className,
+  size,
+  children,
+  disabled,
+  href,
+  ...props
+}) => {
+  return (
+    <a
+      className={classNames(
+        'font-inter text-primary-text font-regular px-xs tracking-normal hover:bg-action-background-hover visited:font-semi-bold',
+        {
+          'text-base leading-24': size !== 'small',
+          'text-14 leading-20': size === 'small',
+          'text-disabled-text hover:bg-background-surface pointer-events-none cursor-not-allowed':
```

Comment:
I will go with `pointer-events-none` as `cursor-not-allowed` won't be much helpful in our case because it is just a visual way to show the not-allowed cursor and doesn't really disable any event that happens on the link unlike `pointer-events-none`

Reply:
I think it's important to have something that visually indicates the link is disabled so rendering a `div` with `a` tag in case it's disabled isn't that big of a price to pay for having a more clear design.

Reply:
yes, I agree with you that we need to have something that visually indicates the link is disabled that is why we have a different style for the disabled link so there is no need to transform the cursor to `not-allowed`